registration-processor/pre-processor/registration-processor-packet-classifier-stage/src/main/java/io/mosip/registration/processor/stages/packetclassifier/utility/PacketClassifierUtility.java

```diff
@@ -0,0 +1,60 @@
+package io.mosip.registration.processor.stages.packetclassifier.utility;
+
+import java.util.List;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import io.mosip.kernel.core.exception.BaseCheckedException;
+import io.mosip.registration.processor.core.exception.util.PlatformErrorMessages;
+
+@Component
+public class PacketClassifierUtility {
+
+	/**
+	 * The mandatory languages that should be used when dealing with field type that
+	 * has values in multiple languages
+	 */
+	@Value("#{'${mosip.mandatory-languages}'.split(',')}")
+	private List<String> mandatoryLanguages;
```

Comment:
What will happen when a empty string is given for mosip.mandatory-languages property, will the list be empty list or list with single entry and the entry is empty string ?

Reply:
list with single entry and that entry will be empty string...

Reply:
Ideally this should be an empty array, can we check if there is a way to achieve this ?

Reply:
Worst case atleast we should remove the empty string using a post construst method.

We cannot have main logic handling and unit test case handling the empty string case, since it is confusing.